dspace-spring-rest/src/main/java/org/dspace/app/rest/model/submit/SelectableRelationship.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model.submit;
+
+public class SelectableRelationship {
```

Comment:
Please add Javadocs to this new class to tell us what a SelectableRelationship is and how/where it is used

Reply:
The JavaDocs have been included